src/catalog/language_catalog.cpp

```diff
@@ -16,13 +16,15 @@
 #include "executor/logical_tile.h"
 #include "storage/data_table.h"
 #include "type/value_factory.h"
+#include "codegen/buffering_consumer.h"
+#include "expression/expression_util.h"
 
 namespace peloton {
 namespace catalog {
 
-LanguageCatalogObject::LanguageCatalogObject(executor::LogicalTile *tuple)
-    : lang_oid_(tuple->GetValue(0, 0).GetAs<oid_t>()),
-      lang_name_(tuple->GetValue(0, 1).GetAs<const char *>()) {}
+LanguageCatalogObject::LanguageCatalogObject(codegen::WrappedTuple tuple)
```

Comment:
Do we want to make a copy of the WrappedTuple parameter? Wouldn't using a reference suffice?